docs/source/reference/config.rst

```diff
@@ -1111,6 +1114,40 @@ Kueue configuration (optional).
 
 Name of the `local queue <https://kueue.sigs.k8s.io/docs/concepts/local_queue/>`_ to use for SkyPilot jobs.
 
+.. _config-yaml-kubernetes-dws:
+
+``kubernetes.dws``
+~~~~~~~~~~~~~~~~~~
+
+GKE DWS configuration (optional).
+
+``kubernetes.dws.mode``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Mode of the DWS (optional), see `GKE DWS documentation <https://cloud.google.com/kubernetes-engine/docs/concepts/dws>`_ for more details.
+
+Can be one of:
+
+- ``flex-start``: Flex start mode. If ``kubernetes.kueue.local_queue_name`` is set, this mode is automatically set to ``flex-start-queued-provisioning``.
+- ``flex-start-queued-provisioning``: Flex start with queued provisioning mode.
+
+Default: ``null``.
```

Comment:
What does it mean by Default: ``null`` for an user, considering you mention that it can only be one of the two options above?